agent/src/main/java/com/appland/appmap/transform/annotations/CtClassUtil.java

```diff
@@ -19,16 +19,13 @@ class CtClassUtil {
    * Otherwise, {@code false}.
    */
   public static Boolean isChildOf(CtClass candidateChildClass, String parentClassName) {
-    // System.err.println("candidateChildClass: " + candidateChildClass.getName() + ", parentClassName: " + parentClassName);
-
```

Comment:
I actually liked these commented-out statements because they indicate to a developer where to poke around when they try to debug this again.